Bump QBUFSZ to cope for large terminals and avoid stack buffer
overflows.

--- include/global.h.orig	Thu Nov 24 03:49:47 2016
+++ include/global.h	Thu Nov 24 03:50:14 2016
@@ -297,7 +297,7 @@ struct version_info {
 #define DOORMAX		120	/* max number of doors per level */
 
 #define BUFSZ		256	/* for getlin buffers */
-#define QBUFSZ		128	/* for building question text */
+#define QBUFSZ		1024	/* for building question text */
 
 #define PL_NSIZ		32	/* name of player, ghost, shopkeeper */
 #define PL_CSIZ		32	/* sizeof pl_character */
